### DISCUSSION TITLE
공모전 관련해서 문의 드립니다2

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#934] [5. Q&A]
제페토 월드 공모전에 참가하려고 하는데 궁금한 점이 있어서 문의 드립니다.

제페토 월드 공모전 참가 신청서를 작성했는데 팀원마다 작성 양식이 달랐습니다.
저는 구글 폼에 팀 이름을 입력하는 부분이 없어서 일단 그냥 제출했는데요.
제가 어떤 팀에 속해있는지 어떻게 알 수 있나요?
그리고 제출 확인 메일을 받지 못했는데 문제 없나요?

팀으로 작업한 경우에도 하나의 월드는 반드시 1명만 등록해야 하나요?
만약 월드에서 수익이 발생하게 되면 어떻게 분배해야 하나요?
월드 크리에이터를 여러명 등록하는 방법은 없는건가요?

알려주세요🙏

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 문의 주신 내용 답변 드립니다. 
공모전 제출 완료 시 따로 메일이 발송되고 있지 않습니다. 아래 두 건을 진행했으면 제출 완료 된 걸로 간주합니다.
- ZEPETO Studio 내 월드 심사 제출 완료
- [구글시트](https://docs.google.com/forms/d/e/1FAIpQLSeBcW3Ie7RcYn9ICKC4DXx39VlpBQD_7bs9JlM8zIgID28NUw/viewform)에 등록

공모전에 한해서는 월드 제출 갯수에 제한은 없습니다.
팀에서 1개만 제출해도 되고, 팀원 모두 제출해도 되고, 팀원 모두 여러개 제출해도 상관은 없습니다.
다만 제출한 월드는 모두 구글시트에 기재해 주셔야 합니다.

수익 쉐어의 경우 현재 시스템 상으로 월드를 제출한 크리에이터 본인 1인에 한해서만 수익 배분이 가능합니다.

감사합니다.
